### PR TITLE
Powersink now wrenchs into the grid instead of being screwdrivered into it.

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

see title
also it gives a message when trying to screwdriver it
update: ominous beacons too

## Why It's Good For The Game

Unsnowflakes the power sink from basically every other thing that can be affixed to the ground. Bolting it down makes more sense than screwing it in.

## Changelog
:cl:
tweak: power sink and ominous beacon now wrenchable, not screwdriverable
/:cl:

### CODE REVIEW

**Comment by kriskog** [comment]
While you are at it, power beacons are the same. :ok_hand:

**Comment by carlarctg** [comment]
il do that 2morrow

**Comment by nemvar** [comment]
Didn't know you could pick up powersinks, guess you learn something new every day.

**Comment by carlarctg** [comment]
yeah theyre bulky

> While you are at it, power beacons are the same. 👌

done. also redid the descriptions for bolting and unbolting to have soul and use src

File: code/game/machinery/syndicatebeacon.dm [hunk 1]
```diff
@@ -53,26 +53,30 @@
...
 		if(anchored)
 			setAnchored(FALSE)
-			to_chat(user, "<span class='notice'>You unscrew the beacon from the floor.</span>")
+			to_chat(user, "<span class='notice'>You unbolt \the [src] from the floor and detach it from the cable.</span>")
 			disconnect_from_network()
 			return
 		else
 			if(!connect_to_network())
-				to_chat(user, "<span class='warning'>This device must be placed over an exposed, powered cable node!</span>")
+				to_chat(user, "<span class='warning'>\the [src] must be placed over an exposed, powered cable node!</span>")
```
Comment (ShizCalev) [inline code/game/machinery/syndicatebeacon.dm]: shouldn't this have been capitalized?

**Pull request merged**